### ISSUE TITLE
Vidcoder 3.12 crashed

### ISSUE THREAD

**Issue by Djfe** [RandomEngy/VidCoder#335]
VidCoder 3.12 just crashed, while clicking on "Built-in" or "General" (somewhere in the preset picker)
``` Anwendung: VidCoder.exe
Frameworkversion: v4.0.30319
Beschreibung: Der Prozess wurde aufgrund einer unbehandelten Ausnahme beendet.
Ausnahmeinformationen: System.InvalidOperationException
   bei System.Windows.Threading.Dispatcher.PushFrame(System.Windows.Threading.DispatcherFrame)
   bei System.Windows.Window.ShowHelper(System.Object)
   bei System.Windows.Window.ShowDialog()
   bei VidCoder.App.OnDispatcherUnhandledException(System.Object, System.Windows.Threading.DispatcherUnhandledExceptionEventArgs)
   bei System.Windows.Threading.Dispatcher.CatchException(System.Exception)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   bei System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   bei MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   bei MS.Win32.UnsafeNativeMethods.MessageBox(System.Runtime.InteropServices.HandleRef, System.String, System.String, Int32)
   bei System.Windows.MessageBox.ShowCore(IntPtr, System.String, System.String, System.Windows.MessageBoxButton, System.Windows.MessageBoxImage, System.Windows.MessageBoxResult, System.Windows.MessageBoxOptions)
   bei System.Windows.MessageBox.Show(System.Windows.Window, System.String, System.String, System.Windows.MessageBoxButton)
   bei VidCoder.Services.PresetsService.TryUpdateSelectedPreset(VidCoder.ViewModel.PresetViewModel)
   bei VidCoder.Controls.PresetTreeViewContainer.TreeView_OnSelectedItemChanged(System.Object, System.Windows.RoutedPropertyChangedEventArgs`1<System.Object>)
   bei System.Windows.RoutedEventArgs.InvokeHandler(System.Delegate, System.Object)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(System.Object, System.Windows.RoutedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(System.Object, System.Windows.RoutedEventArgs, Boolean)
   bei System.Windows.UIElement.RaiseEventImpl(System.Windows.DependencyObject, System.Windows.RoutedEventArgs)
   bei System.Windows.Controls.TreeView.ChangeSelection(System.Object, System.Windows.Controls.TreeViewItem, Boolean)
   bei System.Windows.Controls.TreeViewItem.Select(Boolean)
   bei System.Windows.Controls.TreeViewItem.OnIsSelectedChanged(System.Windows.DependencyObject, System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.FrameworkElement.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.NotifyPropertyChange(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.UpdateEffectiveValue(System.Windows.EntryIndex, System.Windows.DependencyProperty, System.Windows.PropertyMetadata, System.Windows.EffectiveValueEntry, System.Windows.EffectiveValueEntry ByRef, Boolean, Boolean, System.Windows.OperationType)
   bei System.Windows.StyleHelper.ApplyStyleOrTemplateValue(MS.Internal.FrameworkObject, System.Windows.DependencyProperty)
   bei System.Windows.StyleHelper.InvalidateContainerDependents(System.Windows.DependencyObject, MS.Utility.FrugalStructList`1<System.Windows.ContainerDependent> ByRef, MS.Utility.FrugalStructList`1<System.Windows.ContainerDependent> ByRef, MS.Utility.FrugalStructList`1<System.Windows.ContainerDependent> ByRef)
   bei System.Windows.StyleHelper.DoStyleInvalidations(System.Windows.FrameworkElement, System.Windows.FrameworkContentElement, System.Windows.Style, System.Windows.Style)
   bei System.Windows.StyleHelper.UpdateStyleCache(System.Windows.FrameworkElement, System.Windows.FrameworkContentElement, System.Windows.Style, System.Windows.Style, System.Windows.Style ByRef)
   bei System.Windows.FrameworkElement.OnStyleChanged(System.Windows.DependencyObject, System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.FrameworkElement.OnPropertyChanged(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.NotifyPropertyChange(System.Windows.DependencyPropertyChangedEventArgs)
   bei System.Windows.DependencyObject.UpdateEffectiveValue(System.Windows.EntryIndex, System.Windows.DependencyProperty, System.Windows.PropertyMetadata, System.Windows.EffectiveValueEntry, System.Windows.EffectiveValueEntry ByRef, Boolean, Boolean, System.Windows.OperationType)
   bei System.Windows.DependencyObject.SetValueCommon(System.Windows.DependencyProperty, System.Object, System.Windows.PropertyMetadata, Boolean, Boolean, System.Windows.OperationType, Boolean)
   bei System.Windows.Controls.ItemsControl.ApplyItemContainerStyle(System.Windows.DependencyObject, System.Object)
   bei System.Windows.Controls.ItemsControl.MS.Internal.Controls.IGeneratorHost.PrepareItemContainer(System.Windows.DependencyObject, System.Object)
   bei System.Windows.Controls.Panel.GenerateChildren()
   bei System.Windows.Controls.Panel.EnsureGenerator()
   bei System.Windows.Controls.Panel.get_InternalChildren()
   bei System.Windows.Controls.StackPanel.StackMeasureHelper(System.Windows.Controls.IStackMeasure, System.Windows.Controls.IStackMeasureScrollData, System.Windows.Size)
   bei System.Windows.Controls.StackPanel.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei MS.Internal.Helper.MeasureElementWithSingleChild(System.Windows.UIElement, System.Windows.Size)
   bei System.Windows.Controls.ItemsPresenter.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Grid.MeasureCell(Int32, Boolean)
   bei System.Windows.Controls.Grid.MeasureCellsGroup(Int32, System.Windows.Size, Boolean, Boolean, Boolean ByRef)
   bei System.Windows.Controls.Grid.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.Controls.Control.MeasureOverride(System.Windows.Size)
   bei System.Windows.FrameworkElement.MeasureCore(System.Windows.Size)
   bei System.Windows.UIElement.Measure(System.Windows.Size)
   bei System.Windows.ContextLayoutManager.UpdateLayout()
   bei System.Windows.ContextLayoutManager.UpdateLayoutCallback(System.Object)
   bei System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
   bei System.Windows.Media.MediaContext.RenderMessageHandlerCore(System.Object)
   bei System.Windows.Media.MediaContext.RenderMessageHandler(System.Object)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   bei System.Windows.Threading.DispatcherOperation.InvokeImpl()
   bei MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   bei System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   bei System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   bei System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   bei MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   bei System.Windows.Threading.DispatcherOperation.Invoke()
   bei System.Windows.Threading.Dispatcher.ProcessQueue()
   bei System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   bei System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   bei MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   bei MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   bei System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   bei System.Windows.Application.RunDispatcher(System.Object)
   bei System.Windows.Application.RunInternal(System.Windows.Window)
   bei VidCoder.App.Main()
```

This is from the windows event viewer, because none of the buttons were working. (copy log etc.)

**Comment by Djfe** [comment]
The second error:
``` Name der fehlerhaften Anwendung: VidCoder.exe, Version: 3.12.0.0, Zeitstempel: 0x5ad3a2ed
Name des fehlerhaften Moduls: KERNELBASE.dll, Version: 10.0.16299.371, Zeitstempel: 0x6369e29f
Ausnahmecode: 0xe0434352
Fehleroffset: 0x0000000000014008
ID des fehlerhaften Prozesses: 0x27b4
Startzeit der fehlerhaften Anwendung: 0x01d3d8d96d72e610
Pfad der fehlerhaften Anwendung: C:\Program Files\VidCoder-Beta\VidCoder.exe
Pfad des fehlerhaften Moduls: C:\WINDOWS\System32\KERNELBASE.dll
Berichtskennung: bf434f88-3f9a-4e61-bf0b-dbc9ffc8692d
Vollständiger Name des fehlerhaften Pakets: 
Anwendungs-ID, die relativ zum fehlerhaften Paket ist: 
```

**Comment by RandomEngy** [comment]
Should be fixed in 3.13 Beta.
